packages/shared/wallet/src/onboard-context.tsx

```diff
@@ -16,12 +16,18 @@ import {
   NetworkId,
 } from '@notional-finance/util';
 
-const chains = [
+export const chains = [
   {
     id: `0x${NetworkId[Network.ArbitrumOne].toString(16)}`,
     token: 'ETH',
-    label: 'Arbitrum One',
-    rpcUrl: getProviderURLFromNetwork(Network.All),
+    label: 'Arbitrum',
+    rpcUrl: getProviderURLFromNetwork(Network.ArbitrumOne),
+  },
+  {
+    id: `0x${NetworkId[Network.Mainnet].toString(16)}`,
```

Comment:
We can just add another key here like `network: Network.ArbitrumOne` and use it to pass to the `updateNotional` call.